### ISSUE TITLE
Why losses are scaled up?

### ISSUE THREAD

**Issue by Grabber** [dog-qiuqiu/Yolo-FastestV2#35]
lbox *= 3.2
    lobj *= 64
    lcls *= 32
    loss = lbox + lobj + lcls

**Comment by Grabber** [comment]
@dog-qiuqiu
